### DISCUSSION TITLE
[phpstorm] form()->route()

### DISCUSSION THREAD

**Discussion by it-can** [spatie/laravel-html#229] [Q&A]
i have this code in my blade file:

```
{{ html()->form('PUT')->route('item.update') }}
```

usually you can click through the route to end up in the routes.php file, but in this code it doesnt allow to click through in PHPStorm.

this works will allow me to click through though

```
{{ route('item.update') }}
```

Is this not possible with this package?

**Comment by mtkumar82** [comment]
How to pass parameters in  route url, I mean like something like 
->route('roles.destroy', $role->id)
or 
->route('roles.destroy',[ $role->id]) i found no documentation for this to pass url parameters for record update or delete. Please suggest solution for this
